source/core-api-authorizers-sample/chalice/merge_template.json

```diff
@@ -612,6 +612,13 @@
             "UpdateReplacePolicy": "Retain",
             "Properties": {
                 "AccessControl": "LogDeliveryWrite",
+                "OwnershipControls": {
```

Comment:
We should NOT use objectOwnership: ObjectOwnership.OBJECT_WRITER as it overrides default for S3 to S3 use case
(according to quip doc)
Also, another recommendation is to remove the ACL from your code and add the policy instead.

Reply:
This change needs to accommodate S3, CloudFront and VPC access logging. The latest CloudFront documentation states that ACLs must be enabled for its logging after the S3 defaults change. There is no bucket policy alternative for that. 

https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#AccessLogsBucketAndFileOwnership

S3 can use either ACLs or a bucket policy for logging, and VPC logs always use a bucket policy.